fairlearn/reductions/_moments/conditional_selection_rate.py

```diff
@@ -43,6 +43,7 @@ def __init__(self, difference_bound=None, ratio_bound=None, ratio_bound_slack=0.
         super(ConditionalSelectionRate, self).__init__()
         if (difference_bound is None) and (ratio_bound is None):
             difference_bound = _DEFAULT_DIFFERENCE_BOUND
+            self.ratio = 1.0
```

Comment:
I think the branching statement for difference_bound is not none handles setting the ratio if we set difference_bound to the default, so I don't think we need to set it here